app/components/resend-invitation/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '', // tagless component
+
+  store: Ember.inject.service(),
```

Comment:
Ace. I figured there might be something like that. Dem release notes!

I think targeted injections make more sense yeah. But you will need to use `get` again.